pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -2373,6 +1993,19 @@ func TestAssignFlavors(t *testing.T) {
 	}
 }
 
+func updateResources(dest resources.FlavorResourceQuantities, src resources.FlavorResourceQuantitiesFlat) {
```

Comment:
Can we use the pre-existing Unflatten function for this, like `clusterQueue.Usage = tc.clusterQueueUsage.Unflatten()`?

Reply:
I could do this in all but one case - a test breaks unfortunately. But I don't like this approach either. We should clean this up in a future PR - by calling some public method of cache or snapshot to get the values we want, rather than manipulating the internal accounting of the CQs/Cohorts

Created https://github.com/kubernetes-sigs/kueue/issues/2502